pkgs/development/tools/clpm/default.nix

```
@@ -0,0 +1,48 @@
+{ stdenv
+, fetchgit
+, wrapLisp
+, sbcl
+, openssl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "clpm";
+  version = "0.3.5";
+
+  src = fetchgit {
+    rev = "v${version}";
+    url = "https://gitlab.common-lisp.net/clpm/clpm";
+    sha256 = "0jivnnp3z148yf4c2nzzr5whz76w5kjhsb97z2vs5maiwf79y2if";
+    fetchSubmodules = true;
+  };
+
+  buildInputs = [
+    (wrapLisp sbcl)
+    openssl
+  ];
+
+  libssl = openssl.out;
+
+  buildPhase = ''
+    # Libs are copied working directory to avoid error when build script tries to deploy them
+    cp $libssl/lib/libcrypto.so.* .
+    cp $libssl/lib/libssl.so.* .
```

```suggestion
  buildPhase = ''
    # Libs are copied working directory to avoid error when build script tries to deploy them
    ln -s $openssl/lib/libcrypto.so.* .
    ln -s $openssl/lib/libssl.so.* .
```

I think that should work.

---

I can confirm that it's working with symlink too. However I still have to set `libssl = openssl.out` as `$openssl` points to `openssl-1.1.1i-bin/` and libraries are present in `openssl-1.1.1i/`. Openssl is a "multi output package" it seems. Is it ok to replace `cp` with `ln` but keep `libssl`?

---

```
ln -s ${openssl.out}/lib/libssl.so.* .
```

Does that work?

---

Ahh, of course. Thanks for this!